bytes/bytes/raw/file_raw_repository.py

```diff
@@ -62,3 +72,68 @@ def _raw_file_path(self, raw_id: UUID, boefje_meta: BoefjeMeta) -> Path:
 
     def _index(self, raw_id: UUID) -> str:
         return str(raw_id)[: self.UUID_INDEX]
+
+
+class S3RawRepository(RawRepository):
+    def __init__(
+        self,
+        file_middleware: FileMiddleware,
+        bucket_per_org: bool,
+        s3_bucket_prefix: str | None = "OpenKAT-",
+        s3_bucket_name: str | None = "OpenKAT",
+    ) -> None:
+        self.file_middleware = file_middleware
```

Comment:
You probably don't want to expose this attribute; otherwise leave as is

```suggestion
        self._file_middleware = file_middleware
```